rosdep: added python3-docutils

atiderko opened the pull request:
Added python3-docutils dependency for
Debian buster:

https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=python3-docutils

Ubuntu focal:

https://packages.ubuntu.com/search?keywords=python3-docutils&searchon=names

atiderko committed:
rosdep: added python3-docutils

Added python3-docutils dependency for
Debian buster:

https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=python3-docutils

Ubunut focal:

https://packages.ubuntu.com/search?keywords=python3-docutils&searchon=names

tfoote reviewed (changes requested):
Thanks for the contribution of rosdep keys. Please take a moment to update your PR following our rosdep rule [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions). You can add changes to the branch you have submitted from or edit the description. When you make the changes please comment back on this PR so that we get a notification that you've updated it.

In particular I'm sure that this package is available on more than just those two specific platform versions from your links. And is likely available on most if not all the other standard platforms.

atiderko committed:
rosdep: updated python3-docutils dependecies

Updated python3-docutils dependency for

Arch:
https://www.archlinux.org/packages/?sort=&q=docutils&maintainer=&flagged=

Debian:
https://packages.debian.org/search?keywords=python3-docutils&searchon=names&suite=all&section=all

Fedora:
https://apps.fedoraproject.org/packages/python3-docutils

Ubuntu:
https://packages.ubuntu.com/search?keywords=python3-docutils&searchon=names&suite=all&section=all

atiderko commented:
Updated python3-docutils dependency for

Arch:
https://www.archlinux.org/packages/?sort=&q=docutils&maintainer=&flagged=

Debian:
https://packages.debian.org/search?keywords=python3-docutils&searchon=names&suite=all&section=all

Fedora:
https://apps.fedoraproject.org/packages/python3-docutils

Ubuntu:
https://packages.ubuntu.com/search?keywords=python3-docutils&searchon=names&suite=all&section=all

tfoote reviewed (approved):
Thanks for the update. It looks good.
